libshaderc/include/shaderc.h

```diff
@@ -31,6 +32,13 @@ typedef enum {
   shaderc_glsl_tess_evaluation_shader
 } shaderc_shader_kind;
 
+typedef enum {
+  shaderc_target_env_vulkan,
+  shaderc_target_env_glsl,
+  shaderc_target_env_glsl_compat,
```

Comment:
Should say how glsl_compat differs from glsl.   I'm confused, and would have to dig into the glslang code to see the differences again.